role_definitions.tf

```diff
@@ -3,7 +3,7 @@ resource "azurerm_role_definition" "user_created" {
   role_definition_id = each.value.role_definition_id
   name               = each.value.name
   description        = each.value.description
-  scope              = each.value.scope
+  scope              = each.value.scope == null ? "/subscriptions/${data.azurerm_client_config.current.subscription_id}" : each.value.scope
```

Comment:
I want to discuss if we can set the default scope to the workload's resource group id. Are there any downsides to doing that @SanderBlom?

Reply:
This would probably be a better to reduce the scope to the workload resource group. I'll update the PR :)